tools/db_bench_tool.cc

```diff
@@ -2526,6 +2535,15 @@ class Benchmark {
     }
   }
 
+  void GenerateKeyFromIntForSeek(uint64_t v, int64_t num_keys, Slice* key) {
+    GenerateKeyFromInt(v, num_keys, key);
+    if (FLAGS_seek_missing_prefix) {
+      assert(prefix_size_ > 8);
+      char* key_ptr = const_cast<char*>(key->data());
+      key_ptr[8] = '1';
```

Comment:
Is this valid only when we assume the prefix are padded with trailing '0's (which we currently do)? If so, maybe worth mentioning why we choose '1' here to prevent people trying to change it to '0'? It sounds detail, thus no strong opinion.